### DISCUSSION TITLE
Monika after story cannot load clothes for her

### DISCUSSION THREAD

**Discussion by Sheaon13** [Monika-After-Story/MonikaModDev#9597] [QA - code and content creation (technical)]
I keep trying to fix it and I have no idea what I'm doing. I'm not a coder or anything so I absolutely have no idea what to do. I've tried to look at threads with people with similar issues and I have no idea what they're talking about or how to fix it. I do not understand whatsoever. 

This is the issue I am having(I pulled it from the log):

[info]:  reading JSON at 'c:\users\sarah\onedrive\desktop\doki doki literature club\game\mod_assets\monika\j\lordbaaa_pink_shirt.json'...
[!ERROR! T_T]:  version not found

[info]:  reading JSON at 'c:\users\sarah\onedrive\desktop\doki doki literature club\game\mod_assets\monika\j\sirnimblybottoms_acs_heart_choker.json'...
[!ERROR! T_T]:  No JSON object could be decoded

**Comment by Booplicate** [comment]
The jsons are missing required fields, so they cannot be loaded. If you're the creator, you can find more information about the format [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#JSON-format).
